targets/sim/video.py

```diff
@@ -3,7 +3,6 @@
 from litevideo.output.common import *
```

Comment:
Does this sim target actually work at all? If not - add a FIXME at the top (but don't require fixing it in this pull request).

Reply:
It seems that `TARGET=sim` does not work at all:
```
(LX P=sim C=vexriscv R=update_submodules) houen@bakura:~/buildenv/348-bump_litex$ make gateware
mkdir -p build/sim_base_vexriscv/
time python -u ./make.py --platform=sim --target=base --cpu-type=vexriscv --iprange=192.168.100     \
        2>&1 | tee -a /home/houen/buildenv/348-bump_litex/build/sim_base_vexriscv//output.20200330-205713.log; (exit ${PIPESTATUS[0]})
Traceback (most recent call last):
  File "./make.py", line 169, in <module>
    main()
  File "./make.py", line 121, in main
    platform = get_platform(args)
  File "./make.py", line 51, in get_platform
    exec("from platforms.{} import Platform".format(args.platform), globals())
  File "<string>", line 1, in <module>
  File "/home/houen/buildenv/348-bump_litex/platforms/sim.py", line 1, in <module>
    from litex.boards.platforms import sim
ImportError: cannot import name 'sim' from 'litex.boards.platforms' (/home/houen//buildenv/348-bump_litex/third_party/litex/litex/boards/platforms/__init__.py)

real    0m0,155s
user    0m0,144s
sys     0m0,012s